Source/Urho3D/Resource/BackgroundLoader.cpp

```diff
@@ -36,6 +34,8 @@
 namespace Urho3D
 {
 
+#ifdef URHO3D_THREADING_GENERAL
```

Comment:
When enabling threads for a target i would either allocate extra worker(s) for things like `BackgroundLoader` (either going over cpu count or at the expense of WorkQueue worker threads)

Reply:
We cannot reliably allocate worker for BackgroundLoader, so I will keep it disabled for now